python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -346,6 +354,30 @@ async def configure(self, config):
                 DeprecationWarning,
             )
 
+        self.time_on_target = config.time_on_target
+        if config.time_on_target == 0:
+            self.time_on_target = self.compute_time_on_target()
+
+    def compute_time_on_target(self):
+        """Determine the amount of time spent on target so the rotator position
+        can be optimized. Assumed slew time is 3 minutes, and that the read,
+        calculation, and offset totals 20s."""
+
+        self.time_on_target = 180 + self.max_acq_iter * (self.acq_exposure_time + 20)
```

Comment:
Consider replacing this variable here with a local variable instead of the class attribute one. Basically, inside this method replace `self.time_on_target` with just `time_on_target`